ui/js/actions/settings.js

```diff
@@ -41,3 +44,23 @@ export function doSetClientSetting(key, value) {
     },
   };
 }
+
+export function getThemes() {
+  // Themes path
+  const themesPath = `${remote.app.getAppPath()}/dist/themes`;
+
+  // Get all .css files
+  const files = readdirSync(themesPath).filter(function(file) {
+    return extname(file) === ".css";
+  });
+
+  // Get theme name
+  const themes = files.map(function(file) {
+    return file.replace(".css", "");
+  });
+
+  return {
+    type: types.GET_THEMES,
+    data: { themes },
+  };
```

Comment:
@btzr-io this name is probably okay given our lack of standards re: action naming, but @6ea86b96 this makes me think we should establish a convention. We don't really have a naming pattern, nor are we consistent with our verbs (e.g. SUCCESS vs. SUCCEEDED).

I'm a a fan of `noun_verb` as a naming convention, since once you get used to the pattern it facilitates both deducing them and autocompleting them.